doc/plugins/eu.esdihumboldt.hale.doc.user/html/tasks/data/save_transformation.html

```diff
@@ -31,11 +31,10 @@ <h1>Save the transformation result</h1>
 		work based on a XML target schema.
 	</p>
 	<p>
-		Supported data export formats in <i>hale�studio</i> are:
+		Supported data export formats in <i>hale»studio</i> can be viewed on the following page:
 	</p>
 	<ul>
-		<li><a href="../../reference/export/xml_data.html">XML</a></li>
-		<li><a href="../../reference/export/gml_data.html">GML</a></li>
+		<li><a href="http://help.halestudio.org/latest/nav/0_7_2_1">Supported formats for export</a></li>
```

Comment:
Downside with linking to the online help is that there may be a different selection of formats than available in the local version (e.g. when the local version includes the pro plugins).
I could not find any information in the [Eclipse Help](https://help.eclipse.org/latest/topic/org.eclipse.platform.doc.isv/guide/ua_help.htm?cp=2_0_19_1) though, how to link to navigation topics.

I asked ChatGPT about it and it [suggests that there is a notation that allows linking to a topic without help page given its id](https://chat.openai.com/share/f2b61921-f032-4e92-af22-cb35c2293633), but I don't know if it just made that up (I could not find any documentation/example of that).

If that does not work a workaround could be adding an HTML file to the Export topic (which currently only has children but no HTML page), and linking to that.
According to the documentation [here](https://help.eclipse.org/latest/topic/org.eclipse.platform.doc.isv/guide/ua_help_content_child_links.htm?cp=2_0_19_1_1_10) it should be possible to generate links to that child pages for that HTML page, for it to be similar to the current auto-generated representation.

Though to properly test all these things it would be required to run hale studio locally from Eclipse.

Reply:
@stempler Would you please provide the information on how to re-build the documentation. Can I find anywhere documented how to build it? 
I have added the `<!--INSERT_CHILD_LINKS-->` and I would like to try it out.
Thank you

Reply:
@emanuelaepure10 Easiest way is to run hale studio locally from Eclipse and launch the help from the menu (Help -> Help)

Reply:
@stempler In this way the files are not "forced" to be recreated, so the `<!--INSERT_CHILD_LINKS-->` is not considered.